app/src/main/res/layout/fragment_parking.xml

```diff
@@ -42,12 +42,11 @@
 
     <Button
         android:id="@+id/fragment_dialog_accept_button"
-        style="@style/button"
+        style="@style/main_activity_button"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:text="@string/fragment_dialog_accept_button_text"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toEndOf="@+id/fragment_dialog_cancel_button"
```

Comment:
be careful with this, the @+id should be ONLY when an id property is defined, otherwise should be only @id.

This is because when you define an @+id you are generating a new pointer and this is only good when you define an id

Reply:
fixed